### PR TITLE
Update default board configs

### PR DESCRIPTION
**Pull request by rsalveti**

### CODE REVIEW

**Comment by gatek33p3r** [comment]
Checkpatch Suceeded.

**Comment by gatek33p3r** [comment]
Bootloader Build Succeeded.

**Comment by gatek33p3r** [comment]
Application Build Succeeded.

File: prj_nrf52.conf [hunk 1]
```diff
@@ -1,5 +1,7 @@
 CONFIG_FLASH=y
 CONFIG_NETWORKING=y
+
+# Network
 CONFIG_NET_TCP=y
 CONFIG_NET_CONTEXT_SYNC_RECV=n
 CONFIG_NET_L2_BLUETOOTH=y
```
Comment (mike-scott) [inline prj_nrf52.conf]: This and the next line under are now duplicated.

File: prj_stm32f4.conf [hunk 2]
```diff
@@ -1,5 +1,7 @@
 CONFIG_FLASH=y
 CONFIG_NETWORKING=y
+
+# Network
 CONFIG_NET_TCP=y
 CONFIG_NET_CONTEXT_SYNC_RECV=n
 CONFIG_NET_L2_BLUETOOTH=y
```
Comment (mike-scott) [inline prj_stm32f4.conf]: This and the next line under are now duplicated.

File: prj_nrf52.conf (hunk 1 above)
Comment (rsalveti) [inline prj_nrf52.conf]: Indeed, updated.

File: prj_stm32f4.conf (hunk 2 above)
Comment (rsalveti) [inline prj_stm32f4.conf]: Indeed, updated.

**Comment by gatek33p3r** [comment]
Checkpatch Suceeded.

**Comment by gatek33p3r** [comment]
Application Build Succeeded.

**Comment by gatek33p3r** [comment]
Bootloader Build Succeeded.

**Comment by mike-scott** [comment]
LGTM

**Pull request merged**